elytron/src/main/java/org/wildfly/extension/elytron/TrivialAddHandler.java

```diff
@@ -58,12 +58,14 @@
         this.initialMode = checkNotNullParam("initialMode", initialMode);
```

Comment:
@darranl @ropalka Is it ok to keep this elytron change or should it be handled by another Elytron PR?

Reply:
@jmesnil this current change looks Ok to me, the issue was previously @ropalka was removing an 'unused' method that coincidentally a second PR was about to start using.